### PR TITLE
Refactor into single crate with lib and bin parts

### PR DESCRIPTION
**Pull request by danielSanchezQ**

### CODE REVIEW

**Comment by danielSanchezQ** [comment]
This is part of #10

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
+chain-core = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "master" }
+chain-crypto = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "master" }
+chain-ser = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "master" }
+chain-time = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "master" }
+chain-impl-mockchain = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "master" }
+chrono = "0.4"
+itertools = "0.10"
+jcli = { git = "https://github.com/input-output-hk/jormungandr.git", branch = "master" }
+jormungandr-lib = { git = "https://github.com/input-output-hk/jormungandr.git", branch = "master" }
+jormungandr-testing-utils = { git = "https://github.com/input-output-hk/jormungandr.git", branch = "master" }
```
Comment (mzabaluev) [inline Cargo.toml:25]: Is this needed for the library/CLI functionality?

File: README.md [hunk 2]
```diff
@@ -9,10 +9,10 @@ Rust based CLI utility for catalyst operations.
 Build with `cargo build` and run, or run with `cargo run -- PARAMS` as per the examples.
 
 ```shell
-catalyst-toolbox-cli 0.1.0
+catalyst-toolbox 0.1.0
 
 USAGE:
-    catalyst-toolbox-cli.exe [FLAGS] [SUBCOMMAND]
+    catalyst-toolbox.exe [FLAGS] [SUBCOMMAND]
```
Comment (mzabaluev) [inline README.md:15]: This looks Windows-specific :frowning_man:

**Review by mzabaluev — COMMENTED** [review]
Looks good.

File: README.md (hunk 2 above)
Comment (danielSanchezQ) [reply README.md:15]: Actually yes. I think I cleaned it at some point, but probably left some. Thanks!

File: Cargo.toml (hunk 1 above)
Comment (mzabaluev) [reply Cargo.toml:24]: Perhaps it makes sense to move that into jormungandr-lib, not now though.

**Review by mzabaluev — APPROVED** [review]

**Pull request merged**